analyzers/src/SonarAnalyzer.CSharp/Rules/DoNotCallInsecureSecurityAlgorithm.cs

```diff
@@ -60,7 +63,7 @@ private void CheckInvocation(SyntaxNodeAnalysisContext context)
             if (methodSymbol.ReturnType.DerivesFromAny(AlgorithmTypes) ||
                 IsInsecureBaseAlgorithmCreationFactoryCall(methodSymbol, invocation.ArgumentList))
             {
-                reportAllDiagnostics(context, invocation.GetLocation());
+                ReportAllDiagnostics(context, invocation.GetLocation());
```

Comment:
When you touch this class, you could move the `||` operators to the beginning of the line. Some boring ones could be single lined.